docs/user/troubleshooting.md

```diff
@@ -251,7 +251,7 @@ Next, check that the network-operator is running:
 kubectl -n openshift-network-operator get pods
 ```
 
-And retrieve the logs. Note that, on multi-master systems, the operator will perform leader election and all other operators will sleep:
+And retrieve the logs. Note that, on HA control planes, the operator will perform leader election and all other operators will sleep:
```

Comment:
I dislike most abbreviations in docs, so I'd prefer "HA" -> "high-availability".  But I don't want folks spelling out "Uniform Resource Identifier", so there's certainly a gray area somewhere, and maybe HA is closer to "English" than "jargon" than it feels to me ;).